src/viam/components/power_sensor/power_sensor.py

```diff
@@ -68,15 +68,14 @@ async def get_power(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Op
             print("The power is", power, "Watts")
 
         Returns:
-            float: power in watts
+            float: The measurement of the power, represented as a float.
```

Comment:
```suggestion
            float: The power reading in watts.
```
"Represented as a float" is redundant with the type `float` right there, but I think units could be more useful.